src/language_concepts/data_types.md

```diff
@@ -87,6 +87,23 @@ fn main() {
 
 The boolean type is most commonly used in conditionals like `if` expressions and `constrain` statements. More about conditionals is covered in the [Control Flow](./control_flow.md) and [Constrain Statement](./constrain.md) sections.
 
+## The String Type
+
+There's a basic string implementation in Noir, which can be useful for logging, for example. Since circuit inputs need to be known at compile time, the string length for an input must be hardcoded into the circuit, like so:
+
+``` rust
+// **input**
+// field = "hello"
+
+fn main(string: pub str<5>) {
+    let hello : str<5> = ("hello");
```

Comment:
Parenthesis are unnecessary here.

Also, this is a nitpick but if we remove the `pub` from the string parameter then we can remove the type annotation on line 99 as well since users will already have an example of it. Removing the annotation also allows users to see by example that type inference still works for string literals without needing to be told so.